registry/handlers/referrers.go

```diff
@@ -59,13 +64,30 @@ func (h *referrersHandler) GetReferrers(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// extract the artifactType filter.
 	var annotations map[string]string
 	var artifactTypeFilter string
 	if artifactTypeFilter = r.URL.Query().Get("artifactType"); artifactTypeFilter != "" {
 		annotations = map[string]string{
 			v1.AnnotationReferrersFiltersApplied: "artifactType",
 		}
 	}
+
+	// extract the page size info. Users define page size by using the n
+	// query parameter. The default page size is 100.
```

Comment:
Comment needs to be updated.
```suggestion
	// query parameter. The default page size is 10.
```

Reply:
sorry. Updated.